djmoney/forms/fields.py

```diff
@@ -51,7 +52,15 @@ def compress(self, data_list):
                 return Money(*data_list[:2])
         return None
 
+    def clean(self, value):
+        if isinstance(value, MONEY_CLASSES):
+            value = (value.amount, value.currency)
+        return super(MoneyField, self).clean(value)
+
     def has_changed(self, initial, data):  # noqa
+        # Django 1.8 has no 'disabled' attribute
+        if hasattr(self, 'disabled') and self.disabled:
+            return False
```

Comment:
Django 1.8 LTS was actually deprecated as of April 1, 2018: https://www.djangoproject.com/download/

So I should perhaps release this as 0.14.2? I presume nothing experimental entered `master` since 0.14.1 :)

Reply:
Yes, thank you :) We'll drop Django 1.8 in 0.15 :)

Reply:
It's released :)